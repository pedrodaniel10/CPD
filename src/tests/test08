2 5 10000 10