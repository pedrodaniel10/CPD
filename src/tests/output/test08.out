0.72 0.83
0.50 0.50
